azure-pipelines.yml

```diff
@@ -67,21 +67,21 @@ jobs:
         condition: and(succeeded(), eq(variables['build.reason'], 'PullRequest'))
       # build, test and pack the packages
       - task: VSBuild@1
-        displayName: 'Build and pack Community Toolkit'
+        displayName: 'Build Community Toolkit Sample'
```

Comment:
FYI - I had to reorder the Build steps because the packing the NuGet Package requires `CommunityToolkit.Maui.SourceGenerators.dll`.

`dotnet pack` will fail if `CommunityToolkit.Maui.SourceGenerators.dll` doesn't already exist before `dotnet pack` is executed.